src/pages/products/products-nav-clear-filters/products-nav-clear-filters.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Button } from '@material-ui/core';
+import { useStyles } from './products-nav-clear-filters.styles';
+import { productsTranslations } from '../../../translations/product.translations';
+
+import {
+  setCategoryFilter,
+  setColorsFilter,
+  setModelsFilter,
+  setPatternsFilter,
+  setSearchFilter
+} from '../../../redux/products/products.actions';
+
+const { CLEAR_FILTERS } = productsTranslations;
+
+const ProductsNavClearFilters = () => {
+  const styles = useStyles();
+  const dispatch = useDispatch();
+  const filters = useSelector(({ Products }) => Products.filters);
+
+  const { isHotItemFilter } = filters;
```

Comment:
If you want to get `isHotItemFilter` and filter it out from filters because other filters are arrays and this one is a boolean flag, then you can use `const { isHotItemFilter, ...arrayFilters }  = filters`. Because currently when you iterate through `filters` you take `isHotItemFilter` too

Reply:
fixed